sdk/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Copyright 2016-2018, Pulumi Corporation.
```

Comment:
Is this an intentional inclusion? Didn't immediately appear related...

Reply:
Ahhh I see, for the proto tools. Neat. Should it be in the proto subdir?

Reply:
I thought it should here because that directory only has proto files in it, but I don't feel strongly so lets do it your way.